app/views/transitive_dependencies/_dependencies.html.erb

```diff
@@ -0,0 +1,22 @@
+<% if dependencies.present? && @latest_version.indexed %>
+  <div class="dep-full--l">
+    <% dependencies.each do |dependency| %>
+      <% if rubygem = dependency.rubygem %>
+        <% if @gem_deps_names.include?(rubygem.name) %>
+          <div><span class="deps_item deps_item--duplicate"><%= rubygem.name %> <%= dependency.clean_requirements %></span></div>
+        <% else %>
+          <ul class="deps"><li><span id="<%=rubygem.name%>-toggle"><%= link_to rubygem_find_by_name_path(gem_name: rubygem.name,format: :js), remote: true do %>
+          <span class="caret"></span><% end %></span><span id="<%=rubygem.name%>-title"><strong><%= rubygem.name %></strong> <%= dependency.clean_requirements %></span>
```

Comment:
Please remove strong from here. Change in font size looks jumpy.

Reply:
Okay. but I thought the gem name should stand out from the version number. Moreover I used the same way it was used to show dependency in gem web page.
https://github.com/rubygems/rubygems.org/blob/master/app/views/rubygems/_dependencies.html.erb#L9

This is without strong
https://user-images.githubusercontent.com/16337625/60722560-f5b42480-9f4e-11e9-8974-139d440a298e.png

This is with strong
https://user-images.githubusercontent.com/16337625/60722616-18ded400-9f4f-11e9-9d1b-c1a38c8913b3.png

Please let me know if I should remove it.

Reply:
okay. it's not about the use of strong tag then. On clicking the toggle arrow, the font size of the gem name increases. please remove that.